generateAndPush.sh

```diff
@@ -2,10 +2,6 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd $DIR
-cd piwik
-git checkout live
```

Comment:
@Frankie75321 Rather than deleting this, should we change the branch to `5.x-dev` ?

Reply:
That's exactly what I suggested to Thomas but his reply was "I would probably remove those 4 lines as they won’t work anyway and shouldn’t be needed"

Reply:
Okay :+1: